orchestrator.py

```diff
@@ -46,7 +47,11 @@ def on_connect(self, client, userdata, flags, rc):
         # reconnect then subscriptions will be renewed.
         client.subscribe(str(topics.Camera.set_overlay))
         client.subscribe(str(topics.Camera.get_overlays))
-        self.publish_camera_status(dumps({"connected": True}))
+        self.publish_camera_status(
+            dumps(
+                {"connected": True, "ip_address": gethostbyname(gethostname())}
+            )
+        )
```

Comment:
Sadly this doesn't seem to work as expected on linux :( for me it's giving me `127.0.1.1` which is not a useful IP for other devices on the network to use, it's similar to `localhost` I think.

[This method here](https://stackoverflow.com/a/28950776) seems to work well for me! If it works on windows it should be better to go with.